README.md

```diff
@@ -180,7 +180,8 @@ Use React with A-Frame. Share your A-Frame React Components!
 - [SkyIslands VR](http://www.skyislandsvr.com/pages/overwater.html) - Floating islands over water
 - [Voxel Castle](https://www.scenevr.com/scenes/398/view) - Castle created with MagicaVoxel and baked in Blender
 - [Monument Valley Castle](https://www.scenevr.com/scenes/397/view) - Another castle created with MagicaVoxel and baked in Blender
-
+- [Lava Cave Game](http://drawvr.com/lava-cave) (by [@drawvr]) - Don't touch the lava
+- 
```

Comment:
[Lava Cave Game](http://drawvr.com/lava-cave) (by [@drawvr]) - Don't touch the lava